src/utils/file.js

```diff
@@ -38,159 +36,10 @@ export async function createFolderStructure(folder, parentFolderName = '.') {
-    fs.readFile(fileName, { encoding: 'base64' }, function (error, data) {
-      if (error) {
-        process.logger?.error(`Error accessing file: ${fileName}`)
-        return reject(error)
-      }
-      return resolve(data)
-    })
-  })
-}
-
-export async function getSubFoldersInFolder(folderName) {
-  process.logger?.debug(`Getting subfolders in ${folderName}`)
-  return new Promise((resolve, reject) => {
-    fs.readdir(folderName, { withFileTypes: true }, function (error, data) {
-      if (error) {
-        process.logger?.error(`Error listing subfolders in: ${folderName}`)
-        return reject(error)
-      }
-      const subFolders = data.filter((d) => d.isDirectory()).map((d) => d.name)
-      return resolve(subFolders)
-    })
-  })
-}
-
-export async function getFilesInFolder(folderName) {
-  process.logger?.debug(`Getting files in ${folderName}`)
-  return new Promise((resolve, reject) => {
-    fs.readdir(folderName, { withFileTypes: true }, function (error, data) {
-      if (error) {
-        process.logger?.error(`Error listing files in: ${folderName}`)
-        return reject(error)
-      }
-      const files = data.filter((d) => !d.isDirectory()).map((d) => d.name)
-      return resolve(files)
-    })
-  })
-}
-
-export async function getIniFilesInFolder(folderName) {
-  process.logger?.debug(`Getting *.ini files in ${folderName}`)
-  return new Promise((resolve, reject) => {
-    fs.readdir(folderName, { withFileTypes: true }, function (error, data) {
-      if (error) {
-        process.logger?.error(`Error listing *.ini files in: ${folderName}`)
-        return reject(error)
-      }
-      const files = data
-        .filter((d) => !d.isDirectory())
-        .map((d) => d.name)
-        .filter((name) => name.endsWith('.ini'))
-      return resolve(files)
-    })
-  })
-}
-
-export async function createFolder(folderName) {
-  process.logger?.debug(`Creating folder ${folderName}`)
-  return new Promise((resolve, reject) => {
-    fs.mkdir(folderName, { recursive: true }, (error, data) => {
-      if (error) {
-        process.logger?.error(`Error creating folder ${folderName}`)
-
-        return reject(error)
-      }
-
-      resolve(data)
-    })
-  })
-}
-
-export async function createFile(fileName, content) {
-  process.logger?.debug(`Creating file ${fileName}`)
-
-  return new Promise(async (resolve, reject) => {
-    fileName = unifyFilePath(fileName)
-
-    if (fileName.split(path.sep).length > 1) {
-      let folderPath = fileName.split(path.sep)
-      folderPath.pop()
-      folderPath = folderPath.join(path.sep)
-
-      if (!(await folderExists(folderPath))) await createFolder(folderPath)
-    }
-
-    fs.writeFile(fileName, content, function (error) {
-      if (error) {
-        process.logger?.error(`Error creating file ${fileName}`)
-        return reject(error)
-      }
-
-      resolve(content)
-    })
-  })
+export async function listIniFilesInFolder(folderName) {
```

Comment:
this utility should be in `@sasjs/utils`

Reply:
done